cmake/coverage.cmake

```diff
@@ -4,95 +4,102 @@
+                    PATTERNS "${OV_COVERAGE_BASE_DIRECTORY}/*.pb.cc" 
+                             "${OV_COVERAGE_BASE_DIRECTORY}/*.pb.h" 
+                             "${OV_COVERAGE_BASE_DIRECTORY}/*/tests/*" 
+                             "${OV_COVERAGE_BASE_DIRECTORY}/*/tests_deprecated/*" 
+                             "${OV_COVERAGE_BASE_DIRECTORY}/thirdparty/*") # Skip some pb files, tests and thirdparty
+ov_coverage_genhtml(INFO_FILE "openvino"
+                    PREFIX "${OV_COVERAGE_BASE_DIRECTORY}")
 
 # Generate reports
 
-ie_coverage_extract(INPUT "openvino" OUTPUT "inference"
+ov_coverage_extract(INPUT "openvino" OUTPUT "inference"
                     PATTERNS "${OV_COVERAGE_BASE_DIRECTORY}/src/inference/*")
-
-ie_coverage_genhtml(INFO_FILE "inference"
+ov_coverage_genhtml(INFO_FILE "inference"
                     PREFIX "${OV_COVERAGE_BASE_DIRECTORY}")
 
-ie_coverage_extract(INPUT "openvino" OUTPUT "legacy"
-                    PATTERNS "${OV_COVERAGE_BASE_DIRECTORY}/src/common/legacy/*")
-ie_coverage_genhtml(INFO_FILE "legacy"
+ov_coverage_extract(INPUT "openvino" OUTPUT "core"
+                    PATTERNS "${OV_COVERAGE_BASE_DIRECTORY}/src/core/*")
+ov_coverage_genhtml(INFO_FILE "core"
                     PREFIX "${OV_COVERAGE_BASE_DIRECTORY}")
 
-ie_coverage_extract(INPUT "openvino" OUTPUT "hetero_plugin"
-                    PATTERNS "${OV_COVERAGE_BASE_DIRECTORY}/src/plugins/hetero/*")
-ie_coverage_genhtml(INFO_FILE "hetero_plugin"
+ov_coverage_extract(INPUT "openvino" OUTPUT "transformations"
+                    PATTERNS "${OV_COVERAGE_BASE_DIRECTORY}/src/common/transformations/*")
+ov_coverage_genhtml(INFO_FILE "transformations"
                     PREFIX "${OV_COVERAGE_BASE_DIRECTORY}")
 
-ie_coverage_extract(INPUT "openvino" OUTPUT "auto_plugin"
-                    PATTERNS "${OV_COVERAGE_BASE_DIRECTORY}/src/plugins/auto/*")
-ie_coverage_genhtml(INFO_FILE "auto_plugin"
+ov_coverage_extract(INPUT "openvino" OUTPUT "low_precision_transformations"
+                    PATTERNS "${OV_COVERAGE_BASE_DIRECTORY}/src/common/low_precision_transformations/*")
+ov_coverage_genhtml(INFO_FILE "low_precision_transformations"
                     PREFIX "${OV_COVERAGE_BASE_DIRECTORY}")
 
-ie_coverage_extract(INPUT "openvino" OUTPUT "preprocessing"
-                    PATTERNS "${OV_COVERAGE_BASE_DIRECTORY}src/common/preprocessing/*")
-ie_coverage_genhtml(INFO_FILE "preprocessing"
+ov_coverage_extract(INPUT "openvino" OUTPUT "legacy"
+                    PATTERNS "${OV_COVERAGE_BASE_DIRECTORY}/src/common/legacy/*")
+ov_coverage_genhtml(INFO_FILE "legacy"
                     PREFIX "${OV_COVERAGE_BASE_DIRECTORY}")
 
-ie_coverage_extract(INPUT "openvino" OUTPUT "transformations"
-                    PATTERNS "${OV_COVERAGE_BASE_DIRECTORY}/src/common/transformations/*")
-ie_coverage_genhtml(INFO_FILE "transformations"
-                    PREFIX "${OV_COVERAGE_BASE_DIRECTORY}")
+if(ENABLE_INTEL_CPU)
```

Comment:
why hetero under CPU?

Let's make sure all the components are covered here

Reply:
Typo fixed